src/net.h

```diff
@@ -1239,6 +1242,8 @@ inline std::chrono::microseconds PoissonNextSend(std::chrono::microseconds now,
     return std::chrono::microseconds{PoissonNextSend(now.count(), average_interval.count())};
 }
 
+void CaptureMessage(const CAddress& addr, const std::string& msg_type, const Span<const unsigned char>& data, bool is_incoming);
```

Comment:
We don't have guidance on line length in the style guide, but anything over 100 columns is a bit long for my taste. Also consider adding a doxygen comment:

```suggestion
/** Dump timestamp and binary message to file */
void CaptureMessage(const CAddress& addr, const std::string& msg_type,
                    const Span<const unsigned char>& data, bool is_incoming);

```

Reply:
Added the doxygen comment, but I think I'll leave the long line as it is.  Looking through the rest of net.h, I see many declarations longer than 100 characters, and I don't see any that have been broken up the way you're suggesting.

I agree with you that it's not great, but I'm going to appeal to consistency in this case.

Reply:
That's fair!